### ISSUE TITLE
Powering off VM causes UTM to lose JIT support

### ISSUE THREAD

**Issue by xycainoff** [utmapp/UTM#3127]
Powering off running VM causes UTM to lose JIT support (because UTM app crashes?). 
I enable JIT from AltStore but it possible only when iPad connected to Mac with AltStore server on it. After enabling JIT, i can successfully work with VM. VM works fine in background. No problem with “pausing” VM and “unpausing” it as well. But sometimes i need to restart VM or shutdown one VM and start another VM. After that (if i understand correctly) UTM app is closed and i can enable JIT support only after connect to mac. Sorry for bad english.

iPad Pro 11 (2018) non-jailbroken
iOS 15.0 (reproduced this issue on iOS 14 as well)
UTM 2.2.4 installed with AltStore beta.

**Comment by conath** [comment]
This is working as currently intended (iOS). UTM for iOS needs to fully quit on iOS to be able to start another or the same VM again. That means the JIT needs to be re-enabled each time.

Duplicate of #9
